jjb/shell/jquery-core-build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -eux
+# Deletes cache and artefacts of npm, bower, git submodules and jenkins builds
+git clean -dffx
+export COMMIT=${GIT_COMMIT:-$(git rev-parse HEAD)}
+echo "Checking nodejs version"
+which node
+node --version
+echo "Checking npm version"
+which npm
+npm --version
+echo "Installing npm"
+npm install
+"`npm bin`/grunt" custom:slim --filename=jquery.slim.js
+npm run jenkins
+echo "Coping files to codeorigin.jquery.com"
+scp dist/jquery.js jenkins@wp-03.ops.jquery.net:/var/www/codeorigin.jquery.com/git/jquery-git.js
```

Comment:
I have some strong opinions about _not_ running all this through scp, but I think at a minimum, we could probably simplify this. Something like:
`scp dist/jquery{,.min,.slim,.slim.min}.js jenkins@wp-03.ops.jquery.net:/var/www/codeorigin.jquery.com/git/`

Reply:
We'll discuss this change with @mgol

Reply:
I have to echo @MightyNerdEric  on this. Not too fond of the scp, _but_ if you're going to do this please make it a little more compact with a bit of for loop magic.

Reply:
We need to get the files on that host so that they're available for development sandboxes to allow users to check if their issue is by any chance already fixed.

Is there a better way to achieve this than via `scp`? This is just what our existing Jenkins uses but we can change it if there's a better way.